Libraries/Experimental/SwipeableRow/SwipeableListView.js

```diff
@@ -33,7 +33,7 @@ const {PropTypes} = React;
 type Props = {
   bounceFirstRowOnMount: boolean,
   dataSource: SwipeableListViewDataSource,
-  maxSwipeDistance: number,
+  maxSwipeDistance: number | Function,
```

Comment:
May be refine the function signature to `(any, string, string) => number` to ensure it always returns a number.